contract_interface.md

```diff
@@ -0,0 +1,30 @@
+# eWASM Contract Interface (ECI) Specification; Version 0
+
+The eWASM Contract Interface (ECI) specifies the structure of a contract module.
+
+### Wire format
+
+Every contract must be stored in the [WebAssembly Binary Encoding](https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md) format (in short, WASM bytecode).
+
+### Imports
+
+A contract can only import symbols specified in the [Ethereum Environment Interface](./eth_interface.md).
+
+### Exports
+
+A contract must have exactly two exported symbols:
+- `memory`: the shared memory space available for the EEI to write into.
```

Comment:
The memory supplied to instantiate will be copied over to the instance memory. It is not a pointer as far as I see.